apps/communications/dialer/test/marionette/lib/views/dialer_options/views.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+/* global module */
+//view needs its accessor
+var DialerOptionsAccessor = require('./accessors');
+//views will instatiate the other views or accessor?, here should need the one for call log
+
+//var CallLogAccessor = 
+
+//var appRoot = require('app-root-path');
+// TODO Change the path once requireFromApp becomes its own module
+//var fromApp = require(appRoot +
+//  '/shared/test/integration/require_from_app');
+var appRoot = require('app-root-path');
+// TODO Change the path once requireFromApp becomes its own module
+var fromApp = require(appRoot +
+  '/shared/test/integration/require_from_app');
+
+
+function DialerOptionsView(client) {
+  this.client = client;
+
+  this.accessors = new DialerOptionsAccessor(client);
+
+}
+
+DialerOptionsView.prototype = {
+ 
+  goToCallLog: function() {
+    var callLogTab = this.accessors.callLogButton;
+    callLogTab.tap();
+
+    var CallLogView = require('../call_log/views.js');
+    var callLogView = new CallLogView(this.client);
+    callLogView.accessors.waitToAppear();
```

Comment:
I will try that. Here is where I am stuck, I change to that view, and seems that I am on the correct frame (iframe.displayed() returns ok) but cannot select anything it is like the focus is not there. This is what I commented you....will try what you say if not will check it with you again as I do not know what else to try.